AirCasting/SessionsSynchronization/Adapters/Services/SessionUploadService.swift

```diff
@@ -53,7 +53,7 @@ final class SessionUploadService: SessionUpstream {
             } catch {
                 promise(.failure(error))
             }
-            client.requestTask(for: request) { result, request in
+            client.requestTask(for: request) { [self] result, request in
```

Comment:
note that this doesn't change anything in practice. The block still captures `self` strongly